src/chrome/content/rules/StartCom.xml

```diff
@@ -43,7 +39,19 @@ Fetch error: http://startcom.org/ => https://startcom.org/: (28, 'Connection tim
 
   <!-- should mitigate against exploitation of the above exclusions -->
   <securecookie host=".*" name=".*" />
+  
+  <test url="http://startcom.org/" />
+  <test url="http://linux.startcom.org/" />
+  <test url="http://forum.startcom.org/" />
+  
+  <test url="http://www.startssl.com/" />
+  <test url="http://auth.startssl.com/" />
+  <test url="http://www.startssl.net/" />
+  <test url="http://www.startssl.org/" />
+  <test url="http://www.startssl.eu/" />
+  <test url="http://www.startssl.us/" />
 
+  <rule from="^http://startcom\.org/" to="https://www.startcom.org/" />
```

Comment:
@reedy It seems like https://startcom.org/ does not respond (times out), and that is why the rule was disabled by the checker. My rule redirects http://startcom.org/ to https://www.startcom.org/ instead. Thanks!